src/server/attrgetsetprop.cpp

```diff
@@ -198,7 +198,7 @@ void Attribute::get_properties(Tango::AttributeConfig_3 &conf)
 		conf.event_prop.per_event.period = Tango::string_dup((const char *)(DEFAULT_EVENT_PERIOD));
 	else
 	{
-		int per = (int)((double)event_period);
+		int per = event_period;
 		str << per;
```

Comment:
We could even optimize by replacing lines 201 and 202 with:
`str << event_period;`